### ISSUE TITLE
Format breaks subsequent code

### ISSUE THREAD

**Issue by goevexx** [DEVSENSE/phptools-docs#156]
When I format code, which has '?>' in it, all subsequent code is is formatted badly.

```php
<?php

$xmldata .= '?>';

$test = true;

if ($test) {
    echo "hi";
}
``` 
==>
```php
<?php

$xmldata .= '?>';

$test = true;

if ($test) {
echo "hi";
}
```
As you might expect, this behaviour appeared building xml data. (`<?xml version="1.0" encoding="UTF-8"?>`)

**Comment by goevexx** [comment]
This is also the case when using ?> in comments

**Comment by Miloslav** [comment]
Thank you, Nicolas. This was fixed, it should be available in the pre-release version shortly

**Comment by Miloslav** [comment]
Available in `1.13.10350`. Closing
